apiserver/facades/client/cloud/cloud.go

```diff
@@ -21,56 +22,105 @@ import (
 	"github.com/juju/juju/state"
 )
 
-type CloudV1 interface {
+// CloudV3 defines the methods on the cloud API facade, version 3.
+type CloudV3 interface {
```

Comment:
I'm not sure why these interfaces exist - maybe just for documentation purposes? But I think the change you've made to them makes sense, given the signature change.